.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "addon-sdk"]
-	path = addon-sdk
-	url = https://github.com/mozilla/addon-sdk.git
```

Comment:
[Firefox 38ESR doesn't need cfx][1].

See also the Travis log: ESR was able to run the tests.


[1]: https://blog.mozilla.org/addons/2015/02/26/jpm-replaces-cfx-for-firefox-38/